File: 1 - Iniciante/C99/BEE1047.c
#include <stdio.h>

int main(){
	int h_ini, min_ini, h_fin, min_fin;
	int horas, minutos;
	
	//Entrada de dados
	scanf("%d %d %d %d", &h_ini, &min_ini, &h_fin, &min_fin);
	
	//Conferir as horas
	minutos = ((h_fin * 60 + min_fin) - (h_ini*60 + min_ini));
	if(minutos<=0){
		minutos += 24*60;
	}
	horas = minutos/60;
	minutos = minutos%60;
	
	//Saída de dados
	printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", horas, minutos);

    return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1047.py
h1, m1, h2, m2 = list(map(int, input().split(" ")))

total_minutes = (h2*60 + m2) - (h1*60 + m1)
if (total_minutes<=0):
    total_minutes += (24*60)
hours = total_minutes//60
total_minutes = total_minutes % 60

print(f"O JOGO DUROU {hours} HORA(S) E {total_minutes} MINUTO(S)")